bin/improver-percentiles-to-probabilities

```diff
@@ -40,7 +40,7 @@ from improver.utilities.statistical_operations import \
 
 
 def main():
-    """
+    r"""
```

Comment:
After @arh89 explained, this is actually intentional, so that the docstring is raw text, as there are backslashes in the docstring, which python is trying to interpret. This is mentioned in https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring.